pkg/virt-controller/watch/topology/nodetopologyupdater.go

```diff
@@ -78,7 +78,7 @@ func (n *nodeTopologyUpdater) sync(nodes []*v1.Node) *updateStats {
 func calculateNodeLabelChanges(original *v1.Node, requiredFrequencies []int64) (modified *v1.Node, err error) {
 	nodeFreq, scalable, err := TSCFrequencyFromNode(original)
 	if err != nil {
-		log.DefaultLogger().Reason(err).Object(original).Error("Can't determine TSC frequency of the original")
+		log.DefaultLogger().Reason(err).Object(original).Errorf("Can't determine original TSC frequency of node %s", original.Name)
```

Comment:
I read 

```
2. Can we make this log line clearer? Maybe put the node name? It's hard to understand this log without looking at the code, original is just a variable name for the node object.
```

in the referenced issue. `Object(original)` is supposed to uniquely identify in the logs the object via name/namespace. If that is not the case something is off. Adding the name to the error message itself should be redundant and is not a good structured way to figure out the objects involved.

Could you share more context? Just want to avoid that our logger has no issue (and highlight that this change is not good practice).